responsivo.css

```
@@ -13,4 +13,27 @@
     flex-grow: 1;
     margin-right: 10px;
  }
-}
\ No newline at end of file
+
+.welcome_container h1, .welcome_container p {
+ 	margin-left: 5px;
+ 	padding-left: 0px;
+ }
+
+ .btn_welcome {
+ 	margin-left: 5px;
+ }
+}
+
+@media only screen and (max-width: 768px) {
+ .welcome_container h1 {
+	font-size: 28px;
+  line-height: 36px;
+ }
+
+ .welcome_container p {
+  font-size: 18px;
+  padding: 0 0 25px 0;
```

Acho que aqui seria mais adequado `margin` ao invés de `padding` tb
```suggestion
  margin: 0 0 25px 0;
```